ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdatedTest.java

```diff
@@ -491,7 +503,7 @@ private JsonRpcResponse resp(
         new JsonRpcRequestContext(
             new JsonRpcRequest(
                 "2.0",
-                RpcMethod.ENGINE_FORKCHOICE_UPDATED.getMethodName(),
+                RpcMethod.ENGINE_FORKCHOICE_UPDATED_V1.getMethodName(),
```

Comment:
This shouldn't be hard-coded to forkchoiceUpdatedV1. Though the way it's used in the test might mean this doesn't matter so much?

Reply:
Good spot. You're right that it doesn't really matter for these, but I've created a getMethodName template for consistency anyway.

I had done something similar for EngineGetPayload but missed this one.